sycl/test-e2e/ESIMD/performance_tests/BitonicSortK.cpp

```diff
@@ -0,0 +1,15 @@
+//==---------------- BitonicSortK.cpp  - DPC++ ESIMD on-device test --------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// REQUIRES: gpu-intel-dg2
```

Comment:
Let's change this to

```
REQUIRES: gpu-intel-dg2 && level-zero
```

level-zero is the main runtime plugin we use to communicate with the GPU driver. there is also opencl, but it's not preferred and sometimes bugs out, so let's not bother testing there since the runtime shouldn't effect the assembly.